util/GO9PTERM

```diff
@@ -16,8 +16,14 @@ if [ "$(uname)" = "Linux" ] && [ -e /dev/kvm ]; then
 	fi
 fi
 
+# The -usb flag is necessary for the system to recognize usb input
```

Comment:
by the way this level of discussion in a review can be pretty common, especially for your first couple PRs, so do not get discouraged.